search-sentiments-map/routes/sentiment.js

```diff
@@ -0,0 +1,48 @@
+/** Server-side script that uses the Google Cloud Natural Language API to get the sentiment score. */
+
+const express = require('express');
+var router = express.Router();  // Using Router to divide the app into modules.
+const language = require('@google-cloud/language');
+
+var bodyParser = require('body-parser');  
+// Create application/x-www-form-urlencoded parser.
+var urlencodedParser = bodyParser.urlencoded({ extended: false })  
+router.post('/', urlencodedParser, (req, res) => { 
```

Comment:
I think you may need a JSDoc comment here.

Reply:
Generally, I haven't seen JSDoc comments for GET/POST methods online, but let me know if you see otherwise

Reply:
Maybe just add a comment there to describe what is done